pleskdistup/common/src/dpkg.py

```diff
@@ -140,10 +140,10 @@ def process_stdout(line: str) -> None:
         if collect_stdout:
             nonlocal stdout
             stdout.append(line)
-        log.info(f"{apt_get_cmd[0]}: {line}")
+        log.info(f"stdout: {line}")
 
     def process_stderr(line: str) -> None:
-        log.err(f"{apt_get_cmd[0]}: {line}")
+        log.info(f"stderr: {line}")
```

Comment:
Are you sure `\n` at the end of the `line` will not be logged as well? I believe `logging` module does it by default.

Reply:
It will, that's why I've `.rstrip`ped it before calling the callback. I thought you had no issue about this newline?